### DISCUSSION TITLE
Moving Between NetSuite Accounts

### DISCUSSION THREAD

**Discussion by andruwaltringer** [burkybang/SuiteAdvanced-Feedback#68] [Q&A] [answered]
I was just wondering if I am doing something wrong when moving to and from NetSuite accounts. I am having to log back in every time I do this and this is listed as one of the features for this extension. Thanks in advance!

**Comment by burkybang** [comment, marked as answer]
Enable the feature in settings called "Maintain List of Recent NetSuite Accounts". It will ask for permission to use your bookmarks. The first time you visit a NetSuite account after this, a new bookmarks folder will be created on your bookmarks bar called "NetSuite Accounts". This folder can be moved anywhere, but its name must not change. Each time you visit a different NetSuite account, a new bookmark will be added to that folder. Use the bookmarks in that folder to switch accounts.

When using a bookmark from that folder, it will prompt you to log into the NetSuite account like normal, but it will not log you out of any other NetSuite accounts.

The list of recent NetSuite accounts can also be accessed from the home menu in NetSuite.
